src/components/ProfileTree/ProfileTree.js

```diff
@@ -38,6 +38,7 @@ export const ProfileTree = memo(({
   record,
   className,
   dataAttributes,
+  showAsHotLink,
```

Comment:
Just suggestion. For now, it looks like you want to show the whole tree as a hotlink :)
```suggestion
  showLabelsAsHotLink,
```